Document new configuration options for air-Q integration (#31193)
home-assistant/home-assistant.github.io: source/_integrations/airq.markdown

{% include integrations/config_flow.md %}

> 

Reviewer: ```suggestion

During the initial configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.

```

> During the initial configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.

Reviewer: ```suggestion
```
I realize this sentence was already in this position before this PR. But I think it makes sense to put it before the config flow, as a precondition. This is also done in other integrations where credentials are required.

Virus Index uses CO2 as a proxy for potential aerosol load and can be seen as an indicator of ventilation sufficiency (0 %: insufficient ventilation, 100 %: all fine).

Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. You can activate them in the air-Q mobile application under "Advanced settings".

## Additional configuration

> After the integration is initialised, the user can configure any of the following two parameter:

Reviewer: ```suggestion
After the integration has been initialized, the user can configure any of the following two parameters:
```

> - **Show values averaged by the device**. Default: on. In its default configuration, air-Q averages the stream of sensor values, and the strength of this averaging can be configured on the device side (not exposed through the HA). However, this integration allows to switch between polling the averaged and the raw data from the device. To poll noisy sensor readings from the device, switch this feature off.

Reviewer: ```suggestion
- **Show values averaged by the device**. Default: `on`. In its default configuration, air-Q averages the stream of sensor values. The strength of this averaging can be configured on the device side (not exposed through the HA). However, this integration allows to switch between polling the averaged and the raw data from the device. To poll noisy sensor readings from the device, set **Show values averaged by the device** to `off`.
```

> - **Clip negative values**. Default: on. For baseline calibration purposes, certain sensor values may briefly become negative. The default behavior is to clip such values to 0.

Reviewer: ```suggestion
- **Clip negative values**. Default: `on`. For baseline calibration purposes, certain sensor values may briefly become negative. The default behavior is to clip such values to 0.
```
